package.json

```diff
@@ -53,7 +54,8 @@
     "dist:win:docker": "./scripts/buildWindowsWithDocker.sh",
     "dist:all": "run-s dist:posix dist:win",
     "release:manual": "run-s version dist:all",
-    "release": "git fetch --tags && git add CHANGELOG.md && standard-version -a && env-cmd -f ./.env electron-builder -ml && env-cmd -f ./.env electron-builder -w --x64 --ia32",
+    "release:tags": "git fetch --tags && git add CHANGELOG.md && standard-version -a",
```

Comment:
Why do we need to separate this?

Reply:
This is the command to execute to create the release tag (and further push it). This is what triggers the github actions so we don't need the build to be executed locally.

Reply:
OK.